src/main/play-doc/operation/ConsolidatedLogging.md

```diff
@@ -2,15 +2,15 @@
 
 When multiple machines are involved in a cluster, it quickly becomes difficult to view the log files of distributed applications. ConductR allows the logging output of itself and the bundles that it executes to be directed to a "syslog collector". Syslog is a widely used protocol for Unix based machines and is supported by a number of cloud-based log providers, as well as local operating system support.
 
-The syslog collector can send the log messages to any kind of logging solution. The ConductR distribution includes Elasticsearch and Kibana as an opt-in logging infrastructure. How to configure Elasticsearch and Kibana or other popular logging solutions are described in the next sections.
+The syslog collector can send the log messages to any kind of logging solution. ConductR provides bundles for Elasticsearch and Kibana as an opt-in logging infrastructure. How to configure Elasticsearch and Kibana or other popular logging solutions are described in the next sections.
 
 ## Logging Structure
 
 Before discussing the types of logging available to you, it will be useful to understand the nature of ConductR logs. There are 3 types of logs:
 
-1. bundle events;
-2. bundle logs; and
-3. ConductR's own logs.
+1. Bundle events
+2. Bundle logs
+3. ConductR logs
```

Comment:
Ha - that's definitely an Australian English convention that I'm going to find hard to drop. ;-)